index.d.ts

```diff
@@ -4,6 +4,10 @@ declare namespace srcset {
 		width?: number;
 		density?: number;
 	}
+
+	interface StringifyOption {
+		minify?: boolean;
```

Comment:
This needs a doc comment

Reply:
Should I add a `@description` here?

Reply:
Done. The `description` will now show on supported IDE:

![image](https://user-images.githubusercontent.com/40715044/97780400-5db86080-1bbf-11eb-8e80-2a6b946d6511.png)

> Screenshot from me viewing `test.js`.

Reply:
Here's how to format it: https://github.com/sindresorhus/typescript-definition-style-guide